__test__/product.test.js

```diff
@@ -119,6 +119,11 @@ describe('Testing Products service', () => {
       .set('Authorization', `Bearer ${token}`);
     expect(response.statusCode).toBe(200);
   });
+  test('it should return 200 for retrieving list of all products for all users', async () => {
+    const response = await request(app)
+      .get('/api/v1/products/unprotected')
```

Comment:
change this route from  '/api/v1/products/unprotected' to '/api/v1/products/public'  at least